tests/integration/components/sl-textarea-test.js

```diff
@@ -1,19 +1,332 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
+import { skip } from 'qunit';
 import hbs from 'htmlbars-inline-precompile';
 
 moduleForComponent( 'sl-textarea', 'Integration | Component | sl textarea', {
     integration: true
 });
 
-test( 'for attribute value on label matches id of textarea', function( assert ) {
+test( 'Default classes are applied', function( assert ) {
     this.render( hbs`
-        {{sl-textarea label="test label"}}
+        {{#sl-textarea}}
+        {{/sl-textarea}}
     ` );
 
-    const wrapper = this.$( '>:first-child' );
+    assert.ok(
+        this.$( '>:first-child' ).hasClass( 'form-group' ),
+        'Has class "form-group"'
+    );
+
+    assert.ok(
+        this.$( '>:first-child' ).hasClass( 'sl-textarea' ),
+        'Has class "sl-textarea"'
+    );
+
+    assert.ok(
+        this.$( '>:first-child' ).find( 'textarea' ).hasClass( 'form-control' ),
+        'Has class "form-control"'
+    );
+});
+
+test( '"value" property is supported', function( assert ) {
```

Comment:
Text should also be entered into the `textarea` element and confirmed that `this.get( 'value' )` contains it